BAZEL.md

```diff
@@ -361,17 +361,24 @@ detailed information.
     bazel test //ledger-client/ods:tests --nocache_test_results
     ```
 
-- Execute an individual test case in a Scala test-suite
+- Execute a specific Scala test-suite class
 
     ```
-    bazel test //ledger-client/ods:tests_test_suite_src_test_scala_com_digitalasset_ods_slick_SqlUtilsTest.scala
+    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala
     ```
+    
+- Execute a test with a specific name
+
+    ```
+    bazel test//ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t "JDBC Ledger DAO should be able to persist and load contracts without external offset"
```

Comment:
Don't you need two `--test-arg` switches here?

Reply:
indeed! now corrected